charts/dbildungs-iam-ldap/values.yaml

```diff
@@ -114,6 +126,63 @@ customLdifFiles:
     objectClass: organization
     o: SPSH
 
+
+# First the apply.ldif gets applied if entry is not existend, then the entries in modify.ldif overwrite the entries
+customExtraLdifFiles: 
+  apply.ldif: |-
+    dn: cn=lehrer,dc=schule-sh,dc=de
+    cn: lehrer
+    objectclass: groupOfUniqueNames
+    objectclass: top
+    uniquemember:
+
+    dn: cn=schueler,dc=schule-sh,dc=de
+    cn: schueler
+    objectclass: groupOfUniqueNames
+    objectclass: top
+    uniquemember:
+
+    dn: ou=oeffentlicheSchulen,dc=schule-sh,dc=de
+    objectclass: organizationalUnit
+    objectclass: top
+    ou: oeffentlicheSchulen
+
```

Comment:
@Developer - can you tell me whether you still need the Ersatzschulen enty or if it can go? (Currently still set via 1password - but if it should stay it should get defined here). And please check that these are all the entries you need or if an entry is missing

Reply:
I think it would be helpful if it is still there, yes

Reply:
These two entries are in 1Password: 
dn: ou=oeffentlicheSchulen,ou=lehrer,dc=schule-sh,dc=de
objectClass: organizationalUnit
objectclass: top
ou: oeffentlicheSchulen

dn: ou=ersatzschulen,ou=lehrer,dc=schule-sh,dc=de
objectClass: organizationalUnit
objectclass: top
ou: ersatzschulen

So the ou=lehrer for öffentliche Schulen has been removed, so i assume adding it like this wihout the ou=lehrer is right? 

dn: ou=ersatzschulen,dc=schule-sh,dc=de
objectClass: organizationalUnit
objectclass: top
ou: ersatzschulen

Reply:
Sorry, I don't understand where any ou=lehrer has been removed from another ou=oeffentlicheSchulen, in one of the ldif files in 1Password?

Reply:
https://github.com/dBildungsplattform/dbildungs-iam-server/pull/649/files In this PR it says:
Entry 4: ou=oeffentlicheSchulen,dc=schule-sh,dc=de
dn: ou=oeffentlicheSchulen,dc=schule-sh,dc=de
changetype: add
objectclass: organizationalUnit
objectclass: top
ou: oeffentlicheSchulen